src/openfermion/utils/_operator_utils.py

```diff
@@ -412,27 +414,39 @@ def load_operator(file_name=None, data_directory=None):
     """
     file_path = get_file_path(file_name, data_directory)
 
-    with open(file_path, 'rb') as f:
-        data = marshal.load(f)
-        operator_type = data[0]
-        operator_terms = data[1]
-
-    if operator_type == 'FermionOperator':
-        operator = FermionOperator()
-        for term in operator_terms:
-            operator += FermionOperator(term, operator_terms[term])
-    elif operator_type == 'QubitOperator':
-        operator = QubitOperator()
-        for term in operator_terms:
-            operator += QubitOperator(term, operator_terms[term])
+    if human_readable:
+        with open(file_path, 'r') as f:
+            data = f.read()
+            operator_type, operator_terms = data.split(": ")
+
+        if operator_type == 'FermionOperator':
+            operator = FermionOperator(operator_terms)
+        elif operator_type == 'QubitOperator':
+            operator = QubitOperator(operator_terms)
+        else:
+            raise TypeError('Operator of invalid type.')
     else:
-        raise TypeError('Operator of invalid type.')
+        with open(file_path, 'rb') as f:
+            data = marshal.load(f)
+            operator_type = data[0]
+            operator_terms = data[1]
+
+        if operator_type == 'FermionOperator':
+            operator = FermionOperator()
+            for term in operator_terms:
+                operator += FermionOperator(term, operator_terms[term])
+        elif operator_type == 'QubitOperator':
+            operator = QubitOperator()
+            for term in operator_terms:
+                operator += QubitOperator(term, operator_terms[term])
+        else:
+            raise TypeError('Operator of invalid type.')
```

Comment:
It should not be necessary to add terms individually. It should be possible to simply do something like `operator = FermionOperator(some_string)` where `some_string` is probably going to be straight from the file, minus the initial heading which says what type it is.

Reply:
I think this code is constructing the operators as you suggest. It uses the `split` function only to separate the type of the operator from the remaining data, which is stored as a string when the data is loaded from plain text

Reply:
Ah, I was looking at the wrong part of the code. Yes, you did do it that way. Great!